src/docs/development/ui/animations/index.md

```diff
@@ -23,6 +23,10 @@ If you have a specific request,
 [file an issue]({{site.github}}/flutter/website/issues).
 {% endcomment -%}
 
+* [Implicit animations codelab](/docs/codelabs/implicit-animations)<br>
+  This codelab covers how to use implicit animations
```

Comment:
Delete "This codelab".  => Covers how...

Also, this is where I'd put a definition list, not a bullet list with line breaks.

Reply:
> Delete "This codelab". => Covers how...
👍

> this is where I'd put a definition list, not a bullet list with line breaks.
I'm a bit confused, can you explain this suggestion in greater detail? This change adds the `Implicit animations codelab` as an entry in the existing bulleted list -- are you suggesting that we change this page so that it no longer uses bulleted lists? If so, I'd suggest breaking out the edit of this page to a separate PR.

Reply:
I've created https://github.com/flutter/website/issues/3329 for this, will move to a separate PR.